CMakeLists.txt

```diff
@@ -4,6 +4,10 @@ project( googletest-distribution )
 
 enable_testing()
 
+include(GNUInstallDirs)
+
+option(ENABLE_GOOGLETEST_INSTALL "Skip installation" OFF )
```

Comment:
I'll add to that, your commit subject is too long, and commit message (better; thanks!) still has a space before the ")". (These are much more minor nits than fixing the option description, though.)